plugins/checks/check-deploy

```diff
@@ -62,9 +62,17 @@ TIMEOUT="${DOKKU_CHECKS_TIMEOUT:-30}"
 TMPDIR=$(mktemp -d /tmp/CHECKS.XXXXX)
 docker cp $DOKKU_APP_CONTAINER_ID:/app/CHECKS $TMPDIR 2> /dev/null || true
 if [[ ! -s "${TMPDIR}/CHECKS" ]] ; then
-    dokku_log_verbose "CHECKS file not found in container: skipping checks"
+    dokku_log_verbose "CHECKS file not found in container: running simple container check..."
     rm -rf $TMPDIR
-    exit 0
+
+    # simple default check to see if the container stuck around
+    # for more thorough checks, create a CHECKS file
+    DOKKU_DEFAULT_WAIT=$((WAIT+TIMEOUT))
+    dokku_log_info1 "Waiting for $DOKKU_DEFAULT_WAIT seconds ..."
+    sleep $DOKKU_DEFAULT_WAIT
```

Comment:
This mirrors the regular check wait + timeout as best we can and let's folks adjust it using the standard variables. I didn't want to introduce another variable and the default wait time of 5 seconds didn't seem reasonable.